charts/redpanda/templates/secrets.yaml

```diff
@@ -136,3 +136,114 @@ stringData:
   {{- end }}
     # intentional empty line
 {{- end }}
+
+{{- if .Values.statefulset.sideCars.configWatcher.enabled }}
+  {{- $values := .Values }}
+  {{- $sasl := .Values.auth.sasl }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "redpanda.fullname" . }}-config-watcher
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+  {{- with include "full.labels" . }}
+  {{- . | nindent 4 }}
+  {{- end }}
+type: Opaque
+stringData:
+  sasl-user.sh: |-
+    #!/usr/bin/env bash
+    set -e
+    
+    sleep 10 # wait for admin api to start
```

Comment:
https://github.com/redpanda-data/helm-charts/blob/9b20abf81e80b2f265a3e75335c002d2ea9cf5c7/charts/redpanda/templates/statefulset.yaml#L264 

This should be good check for Admin API readiness.

Reply:
Awesome, yes I completely forgot about this, thank you!